neqo-transport/src/connection/tests/handshake.rs

```diff
@@ -768,16 +768,11 @@ fn anti_amplification() {
     assert_eq!(s_init1.len(), PATH_MTU_V6);
     let s_init2 = server.process_output(now).dgram().unwrap();
     assert_eq!(s_init2.len(), PATH_MTU_V6);
-
-    // Skip the gap for pacing here.
-    let s_pacing = server.process_output(now).callback();
-    assert_ne!(s_pacing, Duration::new(0, 0));
-    now += s_pacing;
-
     let s_init3 = server.process_output(now).dgram().unwrap();
     assert_eq!(s_init3.len(), PATH_MTU_V6);
     let cb = server.process_output(now).callback();
-    assert_ne!(cb, Duration::new(0, 0));
+    // We are blocked by the amplification limit now.
+    assert_eq!(cb, Duration::from_millis(300));
```

Comment:
```suggestion
    assert_eq!(cb, DEFAULT_RTT * 3);
```

You want this to be relative to the RTT, I think.